Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:16
+
+WORKDIR /app
+
+COPY . /app
+RUN yarn install
```

Comment:
@jerivas Is there a reason you didn't add any `CMD` here?

Reply:
Oh, good catch. I think we want `yarn serve` in the compose file for development, but what should be the command for production? `yarn build`?

Reply:
At this point I only envision Docker being used for development, so I might start with `yarn serve` in both cases? But `yarn build-prod` is fine too.

Reply:
Add `yarn serve` in 43b979cfd3c865c64ed9211702fed45022a9c605